modules/web-console/e2e/testcafe/mocks/agentTasks.js

```diff
@@ -103,6 +108,12 @@ export const AGENT_DISCONNECTED_GRID = {
     clusters: []
 };
 
+export const INACTIVE_CLUSTER = {
+    hasAgent: true,
+    hasDemo: true,
+    clusters: [_.merge({}, CLUSTER_1, {active: false})]
```

Comment:
You don't need lodash to do this, use `Object.asssign` or  object spread instead.

Reply:
Changed to Object.asssign.